pkg/fanal/analyzer/pkg/apk/apk.go

```diff
@@ -89,7 +95,9 @@ func (a alpinePkgAnalyzer) parseApkInfo(scanner *bufio.Scanner) ([]types.Package
 		case "F:":
 			dir = line[2:]
 		case "R:":
-			installedFiles = append(installedFiles, path.Join(dir, line[2:]))
+			absPath := path.Join(dir, line[2:])
+			patchSystemInstalledFiles(&pkg, []string{absPath}, opts)
+			installedFiles = append(installedFiles, absPath)
```

Comment:
We should merge these two values since `SystemInstalledFiles` is now just a collection of installed files for all packages. It doesn't make sense to have them differently. But we can refactor that after merging the PR.